test/quantization/test_adaround.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch.nn as nn
+from torch.testing._internal.common_quantization import QuantizationTestCase
+from torch.quantization import QuantStub, DeQuantStub
+import torch.quantization._adaround as _adaround
+
+
+class TestAdaround(QuantizationTestCase):
+    def single_layer_adaround(self, model, adaround_func, img_data):
```

Comment:
this is not implemented yet

I think we can compare with default fake quant and check if the sqnr is always better

Reply:
We should compare with default PTQ, not fake-quant.

Reply:
how is default fake quant without learning weight different from PTQ?

Reply:
@edmundw314 pleases make sure when you compare with default fake quant do not make the weight learnable. it should just be observing and fake quant, but without backprop or the optimizer should not have any parameters.